openapi-data-api-v1/preview-bundled.sh

```diff
@@ -0,0 +1,5 @@
+cd "$(dirname "$0")"
+
+BUNDLED_OPENAPI_FILE="../openapi-data-api-v1.bundled.yaml"
+
+npx @redocly/cli@latest preview-docs "$BUNDLED_OPENAPI_FILE"
```

Comment:
rather than have these 2 bash scripts, consider making a node.js project for this.

and then both these scripts can be in a package.json file.

this has the added bonus of being able to use [nodemon](https://www.npmjs.com/package/nodemon) (or smthn similar) to watch these files for changes and auto-update the preview.

Reply:
Sure, switched to node. Don't need nodemon though, the CLI preview already watches your files.